README.md

```diff
@@ -11,10 +11,10 @@ This is a JavaScript library to easily implement a Google Maps map into your Rea
   - [Basic Google Map](./examples/basic-google-map)
   - [Google Map with Markers](./examples/google-map-with-markers)
   - [Places Autocomplete](./examples/places-autocomplete)
-  - [Directions Service](./examples/directions)
-  - [Geocoding Service](./examples/geocoding)
-  - [Places Library](./examples/places)
-  - [Distance Matrix Service](./examples/distance-matrix)
+  - [Directions Service](./examples/directions-service)
+  - [Geocoding Service](./examples/geocoding-service)
+  - [Places Service](./examples/places-service)
+  - [Distance Matrix Service](./examples/distance-matrix-service)
```

Comment:
Could you please sort the examples here the same way I did in the other READMEs? It seems like I forgot this one. 😬